cmake/CMakeCargo.cmake

```diff
@@ -34,4 +76,6 @@ function(cargo_build)
   add_library(${CARGO_NAME} STATIC IMPORTED GLOBAL)
   add_dependencies(${CARGO_NAME} ${CARGO_NAME}_target)
   set_target_properties(${CARGO_NAME} PROPERTIES IMPORTED_LOCATION ${LIB_FILE})
+
+  install(FILES ${LIB_COMPILED_FILE} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
rather

    install(FILES ${LIB_COMPILED_FILE} DESTINATION bin)

is more standard